src/index.js

```diff
@@ -779,6 +779,21 @@ export default class ReactTable extends Methods(Lifecycle(Component)) {
         }}
         {...rootProps.rest}
       >
+        {showPagination.top ? (
```

Comment:
Looking now, I pass the prop, "placement",  to the Pagination Component in order to append either "topPagination" or "bottomPagination" to the className of the root div of the PaginationComponent. That way it's easier to style/select the pagination component.